optashift-employee-rostering-restclient/pom.xml

```diff
@@ -29,7 +29,7 @@
     </dependency>
     <dependency>
       <groupId>org.jboss.resteasy</groupId>
-      <artifactId>resteasy-jackson-provider</artifactId>
+      <artifactId>resteasy-jackson2-provider</artifactId>
```

Comment:
The main pom also references "resteasy-jackson-provider", please change it there too.

Reply:
Done